ci/rat-regex.txt

```diff
@@ -1,6 +1,6 @@
 ^BUILDS$
 ^autom4te\.cache$
-^build$
+^m4$
```

Comment:
Do we want rat to check our m4 files?  I wonder if this line should just be removed? @bneradt do you have any input?

Reply:
RAT checks our CMake scripts. This seems like an analogous case for automake. Just my two cents.

Reply:
Probably good to check them. Let's do that as a separate PR though in case something goes wrong or we change our mind so it would be easy to revert.